src/server/package.json

```diff
@@ -24,7 +24,7 @@
   "scripts": {
     "patch": "ts-node ../pre_release.ts",
     "dev:watch": "NODE_PATH=../ nodemon",
-    "dev": "NODE_PATH=../ ts-node -P ./tsconfig.server.json ./main.ts",
+    "dev": "yarn build && NODE_PATH=./build node ./build/server/main.js",
```

Comment:
this is not an actual dev run, the bug is coming from `ts-node`. We can leave the problem here, I'll solve it in the future

Reply:
i revert this change, but I'm curious how you define dev run